### ISSUE TITLE
Never recieve the verification SMS nor call on a gapp-less ROM

### ISSUE THREAD

**Issue by m93a** [signalapp/Signal-Android#12775]
I have a clean install of crDroid 9.1 (Android 13, based on Lineage) on a Redmi Note 7 (lavender) without gapps or any gapps shim installed. I've been a regular user (and sponsor) of Signal for years, so I'm already registered – however I didn't manage to get a backup from my old device, so I need to re-register in order to start using Signal again.

I've downloaded the [apk from the official website](https://signal.org/android/apk/) and tried to follow the registration procedure. However, I didn't recieve neither the verification SMS, nor the call, and after a few tries I got the infamous _“You've made too many attempts to register this number. Please try again later.”_ message. At the same time, all of my Signal Desktop clients started showing _“Click to relink Signal Desktop to your mobile device to continue messaging.”_, so I'm unable to use Signal to communicate at all.

In other issues here, it has been stated that Signal works just fine without gapps, as long as I don't install a half-functional replacement for them: [[1]](https://github.com/signalapp/Signal-Android/issues/9279#issuecomment-1075344340), [[2]](https://github.com/signalapp/Signal-Android/issues/11445#issuecomment-998838017), [[3]](https://github.com/signalapp/Signal-Android/issues/9729). So I'm left wondering what am I doing wrong.

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Don't recieve the verification SMS when registering on a phone without gapps.

### Steps to reproduce
- install crDroid 9.1 (lavender)
- walk through the initial setup
- enable developer options
- enable USB debugging
- `adb install Signal-Android-website-prod-universal-release-6.9.2.apk`
- `adb logcat > `[`log.txt`](https://github.com/signalapp/Signal-Android/files/10674089/log.txt)
- open Signal and allow all permissions
- “Continue”, enter my phone number, click “Next”

**Actual result:** _“You've made too many attempts to register this number. Please try again later.”_
**Expected result:** Recieving an SMS and being able to re-register.

### Device info
**Device:** Xiaomi Redmi Note 7 (lavender)
**Android version:** 13 (crDroid 9.1)
**Signal version:** 6.9.2

### Link to debug log
[log.txt](https://github.com/signalapp/Signal-Android/files/10674089/log.txt)

**Comment by m93a** [comment]
I tried again with crDroid + MindTheGapps and the verification call worked. But I would love to use my phone without Google Services – that's why I did the reinstall in the first place.

**Comment by greyson-signal** [comment]
Hi there, I just setup a device without play service and was able to register on 6.11.1 (currently in beta). So not quite sure what the issue would be.

**Comment by m93a** [comment]
I tried again on a fresh install with MinMicroG NoGoolag today and I finally recieved the verification call! However, I also realized that I haven't recieved the verification SMS on neither one of my crDroid installs (clean, MindTheGapps, MinMicroG), so the problem had to be somewhere else. And sure enough – replacing the default SMS app with [Simple SMS Messenger](https://f-droid.org/en/packages/com.simplemobiletools.smsmessenger/) fixed the problem, and now I can recieve even the verification SMS.

So the problem wasn't with Signal after all – the reason why I didn't recieve the verification call on a clean ROM was probably just the time limit on how many verification calls can one try in a given time period. Thanks for the help!
